CMakeLists.txt

```diff
@@ -73,11 +73,18 @@ find_package(OpenMP REQUIRED)
 find_package(GLUT REQUIRED)
 find_package(OpenGL REQUIRED)
 find_package(OIIO REQUIRED)
+IF(WIN32)
+    find_package(GLEW REQUIRED)
+ENDIF(WIN32)
```

Comment:
style nit: there might be a mix in usage, but in general please prefer the lowercase `if (WIN32) ... endif()` cmake tokens. The older `IF ... ENDIF` shouting style is only needed by very old cmake versions.

Reply:
I think it's more consistent with code above.
![image](https://user-images.githubusercontent.com/33077106/176699006-982a9833-8c23-4a4f-94d8-38e79930b135.png)